src/core/renderer_gl/renderer_gl.cpp

```diff
@@ -616,18 +615,88 @@ void RendererGL::displayTransfer(u32 inputAddr, u32 outputAddr, u32 inputSize, u
 	}
 
 	auto destFramebuffer = getColourBuffer(outputAddr, outputFormat, outputWidth, outputHeight);
-	Math::Rect<u32> destRect = destFramebuffer.getSubRect(outputAddr, outputWidth, outputHeight);
+	Math::Rect<u32> destRect = destFramebuffer->getSubRect(outputAddr, outputWidth, outputHeight);
+
+	if (inputWidth != outputWidth) {
+		// Helpers::warn("Strided display transfer is not handled correctly!\n");
+	}
 
 	// Blit the framebuffers
-	srcFramebuffer.fbo.bind(OpenGL::ReadFramebuffer);
-	destFramebuffer.fbo.bind(OpenGL::DrawFramebuffer);
+	srcFramebuffer->fbo.bind(OpenGL::ReadFramebuffer);
```

Comment:
Unchecked optional access?

Reply:
optional only matters if createIfNotFound is false, otherwise the function is guarantted to return a surface